src/VendorChecks/test/tstMetis.cc

```diff
@@ -3,18 +3,16 @@
  * \file   VendorChecks/test/tstMetis.cc
  * \date   Wednesday, May 11, 2016, 12:01 pm
  * \brief  Attempt to link to libmetis and run a simple problem.
- * \note   Copyright (C) 2016-2019, Triad National Security, LLC.
- *         All rights reserved. */
+ * \note   Copyright (C) 2016-2020, Triad National Security, LLC., All rights reserved. */
 //------------------------------------------------------------------------------------------------//
 
+#include "metis.h"
```

Comment:
Did clang tidy make this change? I thought the quote syntax was typically for directory local files -- is there some other use case I'm not aware of? :)

Reply:
No, I made that change.  

While there is no right-wrong answer to this question, the angle-brackets are typically reserved for system headers and are processed after all of the headers listed with quotes.  quotes should always be used for files in the current source directory.  TPL are neither local files or system files, so there is some ambiguity in how they should be treaded.  I made this change to so the Metis headers to be loaded before system headers.  I don't have a strong preference here.  If you want me to switch it back I can.

Reply:
Ahh, okay! I think it's fine this way. I was just curious. Your explanation makes sense, thanks!